var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -688,6 +292,13 @@ def cmake_args(self):
 
         if spec.satisfies('@4.0.0:') and spec.satisfies('platform=linux'):
             cmake_args.append('-DCMAKE_BUILD_WITH_INSTALL_RPATH=1')
+
+        # When building flang branch we do not use the mono-repo
+        if '+flang' not in spec:
+            # Semicolon seperated list of projects to enable
+            cmake_args.append(
+                '-DLLVM_ENABLE_PROJECTS:STRING=' + ';'.join(projects))
```

Comment:
So, if `+flang` is in the spec then how are the various "projects" specified by the variants enabled or disabled?

Reply:
Other than `openmp` and `clang` the other projects are not enabled for the Flang fork.  The Flang fork is likely to have compatibility issues with the other tools `compiler-rt`, `libcxx` etc.  This is what the current package does, it only has resources for `flang-driver` and `openmp` for the flang versions.  

`openmp` and `clang` and enabled by being placed within the flang fork directory and are enabled automatically by being in the old layout.  The `openmp` and `clang` resources are included when `+flang`, which is confusing,  `+flang~openmp` will still build `openmp`.  

From what I understand the flang version/variant is meant to be a dependency package for flang, and not built alone.